system/service/audit_service.py

```diff
@@ -49,12 +49,87 @@ def create_audit(self, document_uid: str, auditor_username: str) -> Optional[Dic
             )
             new_audit = self.audit_repo.create_audit(audit)
+            current_app.logger.info(f"New audits record {new_audit}")
             return {"auditUid": new_audit.uid}
-        return None
\ No newline at end of file
+        return None
+
+    def get_audit_result(self, document_uid: str) -> Optional[Dict]:
+        """Retrieve audit result by document UID."""
+        audit = self.audit_repo.get_audit_by_document_uid(document_uid)
+        if audit:
+            audit_status = self.audit_repo.get_audit_status_by_audit_status_id(audit.audit_status_id)
+            if audit_status:
+                audit_status_value = audit_status.audit_status_value
+                current_app.logger.info(f"audit_result is {audit_status_value}")
+                if audit_status_value == 1:
+                    auditor = self.user_repo.find_user_by_username(audit_status.name)
+                    audit_result = {
+                        'auditUid': audit.uid,
+                        'documentUid': audit.document_id,
+                        'auditStatus': audit_status.audit_status_value,
+                        'auditor': auditor,
+                        'auditedTime': audit_status.updated_date.isoformat() + 'Z'
+                    }
+                elif audit_status_value == 2:
+                    auditor = self.user_repo.find_user_by_username(audit_status.name)
+                    audit_result = {
+                        'auditUid': audit.uid,
+                        'documentUid': audit.document_id,
+                        'auditStatus': audit_status.audit_status_value,
+                        'rejectedReason': audit.rejected_reason,
+                        'auditor': auditor,
+                        'auditedTime': audit_status.updated_date.isoformat() + 'Z'
+                    }
+                elif audit_status_value == 3:
+                    auditor = self.user_repo.find_user_by_username(audit_status.name)
+                    audit_result = {
+                        'auditUid': audit.uid,
+                        'documentUid': audit.document_id,
+                        'auditStatus': audit_status.audit_status_value,
+                        'auditor': auditor
+                    }
+                else:
+                    audit_result = None
+                    current_app.logger.info(f"Error! audit_result is not match 1, 2, 3.")
+                return audit_result
+            else:
+                current_app.logger.info(f"Error! Cannot get audit_status by audit_status_id")
+                return None
+        else:
+            current_app.logger.info(f"Error! Cannot get audit by document_uid")
+            return None
+    
+    def submit_audit_result(self, document_uid: str, audit_uid: str, audit_status: int, rejected_reason: str):
+        """Send audit result by document UID."""
+        audit = self.audit_repo.get_audit_by_document_uid(document_uid)
```

Comment:
typo:
```suggestion
        audit = self.audit_repo.get_audit_by_document_id(document_uid)
```

Reply:
Solved in [1fc3d62](https://github.com/NYCU-CloudNativeTeam/DocumentSystem/pull/1/commits/1fc3d6226434c0883ec375afc6e7fa8d45a12d3b)